src/server.ts

```diff
@@ -251,7 +251,7 @@ export default class AtomsServer {
         debugInfo(`✅: Building minified CSS bundle => ${bundleCssMinPath}`);
       }
     } catch (err) {
-      console.error(chalk.red(err));
+      console.error(err);
```

Comment:
Chalk was messing with the stack trace. Now we will print stack track when any of the promise fails.